nixos/tests/gitlab.nix

```diff
@@ -40,10 +36,10 @@ in {
       virtualisation.useNixStoreImage = true;
       virtualisation.writableStore = false;
 
-      systemd.services.gitlab.serviceConfig.Restart = mkForce "no";
-      systemd.services.gitlab-workhorse.serviceConfig.Restart = mkForce "no";
-      systemd.services.gitaly.serviceConfig.Restart = mkForce "no";
-      systemd.services.gitlab-sidekiq.serviceConfig.Restart = mkForce "no";
+      systemd.services.gitlab.serviceConfig.Restart = lib.mkForce "no";
+      systemd.services.gitlab-workhorse.serviceConfig.Restart = lib.mkForce "no";
+      systemd.services.gitaly.serviceConfig.Restart = lib.mkForce "no";
+      systemd.services.gitlab-sidekiq.serviceConfig.Restart = lib.mkForce "no";
```

Comment:
inherit mkForce from lib

Reply:
Bikeshedding Battle.

I wanna invoke @supersandro2000.

What is better?

- add the `lib.` prefix in each `lib` function 

- inherit them on the top of file (or, for that matter, in any place before they are used)

Reply:
Prefixing lib here is totally fine IMO and better than maintaining an inherit list. In question just remove this change and we can merge the other things.